xscen/io.py

```diff
@@ -518,6 +488,29 @@ def coerce_attrs(attrs):
             raise
 
 
+def _rechunk_for_saving(ds, rechunk):
+    """Do chunking before saving to .zarr or .nc, generalized as Y/X for different axes lat/lon, rlat/rlon"""
```

Comment:
```suggestion
    """Rechunk before saving to .zarr or .nc, generalized as Y/X for different axes lat/lon, rlat/rlon."""
```
To satisfy the nitpicky pre-commit check.